components/ws-manager/pkg/manager/create.go

```diff
@@ -432,7 +432,7 @@ func (m *Manager) createDefiniteWorkspacePod(startContext *startWorkspaceContext
 						MountPropagation: &mountPropagation,
 					},
 				)
-				pod.Spec.Containers[i].Command = []string{pod.Spec.Containers[i].Command[0], "ring0"}
+				pod.Spec.Containers[i].Command = []string{"/.supervisor/workspacekit", "ring0"}
```

Comment:
Is this for all types of workspaces from the moment we deploy?

Reply:
All types that use user-namespaces. Basically we're tying user namespaces to registry facade:
- `registry_facade` FF can run by itself
- `user_namespace` FF mandates the `registry_facade` FF